package.json

```diff
@@ -28,7 +28,9 @@
     "example:send-onetimekey-tx": "ts-node examples/send-onetimekey-tx",
     "example:consume": "ts-node examples/consume",
     "example:register-contract": "ts-node examples/register-contract",
-    "example:generate-account": "ts-node examples/generate-account"
+    "example:generate-account": "ts-node examples/generate-account",
+    "example:register-chain": "ts-node examples/register-chain",
+    "example:register-faucet": "ts-node examples/register-faucet"
   },
```

Comment:
Could you generate the docs locally running `npm run generate:docs` and checking that the generated Doc is what you expect? For example `HttpClient` should not be part of the docs as it's purely internal. You can also verify the descriptions and stuff like that are well written

Reply:
I see, good catch! I would need to add the following annotation onto the internal classes
```
/**
 * @hidden
 */
```